command/operator_diagnose.go

```diff
@@ -288,4 +290,37 @@ func (c *OperatorDiagnoseCommand) offlineDiagnostics(ctx context.Context) error
 		}
 		return nil
 	})
+
+	diagnose.Test(ctx, "disk-usage", func(ctx context.Context) error {
+		partitions, err := disk.Partitions(false)
+		if err != nil {
+			return err
+		}
+
+		partitionExcludes := []string{"/boot"}
+	partLoop:
+		for _, partition := range partitions {
+			for _, exc := range partitionExcludes {
+				if strings.HasPrefix(partition.Mountpoint, exc) {
+					continue partLoop
+				}
+			}
+			usage, err := disk.Usage(partition.Mountpoint)
+			testName := "disk-usage: " + partition.Mountpoint
+			if err != nil {
+				diagnose.Warn(ctx, fmt.Sprintf("could not obtain partition usage for %s", partition.Mountpoint))
```

Comment:
do we want to report the error in here? i think that may be helpful?

Reply:
Yep, good call